### PR TITLE
Import getYAMLConfig util

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
[Import getYAMLConfig util](https://trello.com/c/dBkEER1C/640-import-getyamlconfig-util)

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8685/

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "url": "https://github.com/juju/juju-gui.git"
   },
   "dependencies": {
+    "babel-plugin-rewire": "^1.1.0",
```
[the remark this answers is not in the document]
Comment (huwshimi) [reply package.json]: Proxyquire is good for modifying nested requires, but not so good for the actual testing. In fact proxyquire was breaking the js build if included in most modules.

I'm hoping we can eventually phase out proxyquire, just too big a job for this branch.

[the remark this answers is not in the document]
Comment (hatched) [reply package.json]: Ahh ok thanks

**Comment by hatched** [comment]
Thanks @huwshimi :shipit:

**Pull request merged**